Library/Homebrew/help.rb

```diff
@@ -17,21 +17,21 @@ module Help
     HOMEBREW_HELP = <<~EOS
       Example usage:
         brew search [TEXT|/REGEX/]
```

Comment:
In the other cases, `[ ]` seem to denote that the argument is optional, so this probably shouldn't have them right?

```suggestion
        brew search TEXT|/REGEX/
```

Also, would it be better to include quotes here for the argument?

Reply:
I think we've used `<>` for optional and `[]` for mandatory (or perhaps the other way around)

Reply:
I think it's the other way around. Not quite sure where I picked it up, but I believe `[]` is also more commonly used to indicate optional arguments.

Reply:
I definitely associate `[]` with optional. That's what we use in command usage strings, too.

We use `<>` to indicate mandatory because it underlines the text it surrounds when put through the formatter. See e.g. `brew help install` and `brew help upgrade`.

Good catch, though, @nandahkrishna